kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,65 @@
+### Domain lifecycle sample scripts
+The WebLogic Server Kubernetes Operator project provides a set of sample scripts to shut-down or start-up a specific managed-server or cluster in a deployed Domain or the entire deployed Domain. Please note that Domain must have been created and deployed before these scripts can start or stop managed servers, clusters or the Domain. These can be helpful when scripting the lifecycle of a WebLogic Server Domain. Please see [Domain Life Cycle](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle) to learn how to start, stop, restart, and scale the WebLogic Server instances in your Domain.
+
+#### Scripts to start and stop a managed server
+The `startServer.sh` script starts a managed server by patching 'spec.managedServers.<server-name>.serverStartPolicy' attribute of the domain resource to `ALWAYS`. It also increases the replica count value for the managed server's cluster by `1`. The script provides an option to keep the replica count value constant. Please see script `usage` information using `-h` option.
```

Comment:
Please fully qualify the replica count attribute (just like you did for the serverStartPolicy attribute).